.github/workflows/dependencies-and-licenses.yml

```
@@ -11,63 +11,117 @@ jobs:
+            cli
+            configuration-service
+            distributor
+            go-sdk
+            helm-service
+            jmeter-service
+            lighthouse-service
+            mongodb-datastore
+            remediation-service
+            secret-service
+            shipyard-controller
+            statistics-service
+            webhook-service
+            resource-service
+          )
+          
+          cd ./keptn || return
+          
+          for MODULE in "${MODULES[@]}"; do
+            echo "🔍 Analyzing dependencies in module $MODULE"
+            cd ./"$MODULE" || return
+            go mod tidy
+            go list -m -json all | \
+              go-licence-detector \
+                -depsTemplate=../.dependencies/templates/dependencies.csv.tmpl \
+                -depsOut="${TMP_DIR}"/"${MODULE}"-dependencies.txt \
+                -overrides=../.dependencies/overrides/overrides.json
+            cd ..
           done
-
+          
+          cd ..
+          
           echo "🔍 Analyzing dependencies in go-utils"
-          ( cd keptn/go-utils || return ; go mod tidy > /dev/null 2>&1; go list -m -json all | go-licence-detector -depsTemplate=../keptn/.dependencies/templates/dependencies.csv.tmpl -depsOut="${TMP_DIR}"/go-utils-dependencies.txt )
-
+          cd ./go-utils || return
+          go mod tidy
+          go list -m -json all | \
+            go-licence-detector \
+              -depsTemplate=../keptn/.dependencies/templates/dependencies.csv.tmpl \
+              -depsOut="${TMP_DIR}"/go-utils-dependencies.txt
+          cd ..
+          
           echo "🔍 Analyzing dependencies in kubernetes-utils"
-          ( cd keptn/kubernetes-utils || return ; go mod tidy > /dev/null 2>&1; go list -m -json all | go-licence-detector -depsTemplate=../keptn/.dependencies/templates/dependencies.csv.tmpl -depsOut="${TMP_DIR}"/kubernetes-utils-dependencies.txt )
-
+          cd ./kubernetes-utils || return
+          go mod tidy
+          go list -m -json all | \
+            go-licence-detector \
+              -depsTemplate=../keptn/.dependencies/templates/dependencies.csv.tmpl \
+              -depsOut="${TMP_DIR}"/kubernetes-utils-dependencies.txt
+          cd ..
+          
           cat "$TMP_DIR"/*.txt | sort | uniq > dependencies-and-licenses-go.txt
-
+          
           echo
           echo "👍 done. written results to ./dependencies-and-licenses-go.txt"
-
+          
           cat dependencies-and-licenses-go.txt
+
       - name: Node dependencies and licenses
         run: |
           echo "🔍 Analyzing dependencies in module bridge"
-          ( cd keptn/keptn/bridge || return ; license-report --only-prod --output=csv > ../../../dependencies-and-licenses-node.txt )
+          cd ./keptn/bridge || return
+          yarn install --frozen-lockfile
+          synp --source-file ./yarn.lock
+          license-report --only-prod --output=csv > ../../dependencies-and-licenses-node.txt
```

```suggestion
          license-report --only=prod --output=csv > ../../dependencies-and-licenses-node.txt
```
From the doc: https://www.npmjs.com/package/license-report